### PR TITLE
Add support for "-" and "." in ticker symbol

### PR DESCRIPTION
**Pull request by sajclarke**
Fix UI: Add support for "-" and "." in addition to letters and numbes in the Token Symbol

### CODE REVIEW

File: src/validate.js [hunk 1]
```diff
@@ -28,7 +28,7 @@ export const gt = (x: number) => (value: ?number) =>
   value && value <= x ? `Must be greater than ${x}.` : null
 
 export const alphanumeric = (value: ?string) =>
-  value && !/^[a-z0-9]+$/i.test(value) ? 'Invalid characters.' : null
+  value && !/^[a-z0-9./-]+$/i.test(value) ? 'Invalid characters.' : null
```
Comment (bshevchenko) [inline src/validate.js]: you should create separate validator for this. alphanumeric means only letters and numbers

**Review by bshevchenko — CHANGES_REQUESTED** [review]

File: src/validate.js (hunk 1 above)
Comment (sajclarke) [reply src/validate.js]: Fixed

File: src/validate.js [hunk 2]
```diff
@@ -30,6 +30,9 @@ export const gt = (x: number) => (value: ?number) =>
 export const alphanumeric = (value: ?string) =>
   value && !/^[a-z0-9]+$/i.test(value) ? 'Invalid characters.' : null
 
+export const regex = (expr: any, value: ?string, error_msg: string) =>
```
Comment (bshevchenko) [inline src/validate.js]: We don't use snake case for variables names...
It (and a lot of other code style remarks) was already mentioned in the previous PRs many times...
If you don't have enough attention to notice such things, then at least add necessary rules to our eslint plugins

**Review by bshevchenko — CHANGES_REQUESTED** [review]

File: src/validate.js (hunk 2 above)
Comment (sajclarke) [reply src/validate.js]: Fixed

File: src/validate.js [hunk 3]
```diff
@@ -30,6 +30,9 @@ export const gt = (x: number) => (value: ?number) =>
 export const alphanumeric = (value: ?string) =>
   value && !/^[a-z0-9]+$/i.test(value) ? 'Invalid characters.' : null
 
+export const regex = (expr: any, value: ?string, errorMessage: string) =>
```
Comment (bshevchenko) [inline src/validate.js:33]: expr should be RegExp, not any

**Review by bshevchenko — CHANGES_REQUESTED** [review]

**Pull request closed without merging**